File: app/src/main/java/com/cn29/aac/binding/ImageBindingUtil.java
package com.cn29.aac.binding;

import android.databinding.BindingAdapter;
import android.widget.ImageView;
import com.squareup.picasso.Picasso;
import de.hdodenhof.circleimageview.CircleImageView;

/**
 * Created by charlesng0209 on 10/9/2017.
 */

public class ImageBindingUtil {

  @BindingAdapter({"imageUrl"})
  public static void loadImage(ImageView view, String url) {
    Picasso.with(view.getContext()).load(url).into(view);
  }

  @BindingAdapter({"imageUrl"})
  public static void loadCircleImage(CircleImageView view, String url) {
    Picasso.with(view.getContext()).load(url).into(view);
  }
}

File: app/src/main/java/com/cn29/aac/binding/ImageBindingUtil.kt
package com.cn29.aac.binding

import android.widget.ImageView
import androidx.databinding.BindingAdapter
import com.squareup.picasso.Picasso
import de.hdodenhof.circleimageview.CircleImageView

/**
 * Created by charlesng0209 on 10/9/2017.
 */
object ImageBindingUtil {
    @JvmStatic
    @BindingAdapter("imageUrl")
    fun loadImage(view: ImageView,
                  url: String?) {
        Picasso.with(view.context).load(url).into(view)
    }

    @BindingAdapter("imageUrl")
    fun loadCircleImage(view: CircleImageView,
                        url: String?) {
        Picasso.with(view.context).load(url).into(view)
    }
}